azure-pipelines.yml

```diff
@@ -250,6 +254,58 @@ stages:
         inputs:
           pathtoPublish: '$(Build.ArtifactStagingDirectory)'
 
+
+  - stage: UnitTestJackett
+    displayName: Unit Tests
+    jobs:
+    - job: UnitTests
+      workspace:
+        clean: all
+      strategy:
+        matrix:
+          Mono:
+            buildDescription: Mono
+            imageName: ubuntu-latest
+            framework: net461
```

Comment:
Could you add net461 as variable?

Reply:
I disagree. .NET full framework is end of life and this won't change. If we make this a variable, ubuntu-latest should a variable etc.